app/src/main/res/values/strings.xml

```diff
@@ -23,4 +23,14 @@
     <string name="Authentication_Card_Driver">Driver</string>
 
     <string name="Authentication_Greeter">Welcome!</string>
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
+
+    <string name="Acknowledgement_Action">Dismiss</string>
+
+
+    <string name="mapbox_access_token" tools:ignore="UnusedResources" translatable="false">
+            pk.eyJ1Ijoidm9uY2hleiIsImEiOiJjbHhka2NhYzQwNnY4MmpwemVkeGs3M2NsIn0.bQZ5LsR753Fvq-AjPxalkg
```

Comment:
This one as well, can you confirm that this one is not sensitive?

Reply:
Yes, thats how they wanted me to set that up xd